config/routes.rb

```diff
@@ -96,6 +96,7 @@
         get :revenue_by_payment_method
         get :retail
         get :unfulfilled_orders
+        get :advanced_details
```

Comment:
note that it is advance_sales not advanced_sales.  names are important.  please change throughout